CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ## DFX
 
+### feat: Initialise the nns with an account controlled by a secp256k1 key
+
+This enables easy access to toy ICP using command line tools.
```

Comment:
Can you also add an explanation how those ICP can be accessed?

Reply:
Done.  The docs are in a separate PR; I will have to add the docs there.